resources/assets/uad_lists.json

```
@@ -2517,7 +2517,7 @@
   },
   "com.huawei.meetime": {
     "list": "Oem",
-    "description": "MeeTime (https://consumer.huawei.com/en/support/content/en-us00956296/). Voice and video calling application by Huawei. Only workds with EMUI 9.1+ Huawei phones. Huawei claims they use E2EE but this wasn't verified and there is not Whitepaper so don't trust them. They also collect metadata. There is no advantages to use this app instead of the reputed open-source Signal app.",
+    "description": "MeeTime (https://consumer.huawei.com/en/support/content/en-us00956296/).\nVoice and video calling application by Huawei. Only works with EMUI 9.1+ Huawei phones.\nHuawei claims they use E2EE but this wasn't verified and there is no Whitepaper so don't trust them.\nThey also collect metadata. There are no advantages to use this app instead of the reputed open-source Signal app.",
```

```suggestion
    "description": "MeeTime (https://consumer.huawei.com/en/support/content/en-us00956296/).\nVoice and video calling application by Huawei. Only works with EMUI 9.1+ Huawei phones.\nHuawei claims they use E2EE but this wasn't verified and there is no whitepaper, so don't trust them.\nThey also collect metadata.",
```
We no longer provide alternatives.